SPECS/fribidi/fribidi.spec

```diff
@@ -0,0 +1,283 @@
+Summary:        Library implementing the Unicode Bidirectional Algorithm
+Name:           fribidi
+Version:        1.0.9
+Release:        2%{?dist}
+License:        LGPLv2+ AND UCD
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/fribidi/fribidi/
+Source:         https://github.com/%{name}/%{name}/releases/download/v%{version}/%{name}-%{version}.tar.xz
+Patch0:         %{name}-drop-bundled-gnulib.patch
+BuildRequires:  gcc
+BuildRequires:  meson
+
+%description
+A library to handle bidirectional scripts (for example Hebrew, Arabic),
+so that the display is done in the proper way; while the text data itself
+is always written in logical order.
+
+%package devel
+Summary:        Libraries and include files for FriBidi
+Requires:       %{name}%{?_isa} = %{version}-%{release}
+
+%description devel
+Include files and libraries needed for developing applications which use
+FriBidi.
+
+%prep
+%autosetup -p1
+
+%build
+%meson -Ddocs=false
+%meson_build
+
+%check
+%meson_test
+
+%install
+%meson_install
+find %{buildroot} -type f -name "*.la" -delete -print
+
+%ldconfig_scriptlets
+
+%files
+%doc README AUTHORS ChangeLog THANKS NEWS TODO
+%license COPYING
+%{_bindir}/fribidi
+%{_libdir}/libfribidi.so.0*
+
+%files devel
+%{_includedir}/fribidi
+%{_libdir}/libfribidi.so
+%{_libdir}/pkgconfig/*.pc
+#%%{_mandir}/man3/*.gz
+
+%changelog
+* Thu Apr 15 2021 Henry Li <lihl@microsoft.com> - 1.0.9-2
+- Initial import from Fedora 32 (license: MIT)
+- Remove distro check that does not apply to CBL-Mariner
```

Comment:
Please check the license, update the `License` tag and the `%license` macro (if needed), and add the phrase `License verified` to the changelog once that's done.

Reply:
Fixed